src/lib/Components/Markdown.tsx

```diff
@@ -66,6 +68,7 @@ export const defaultRules = {
   },
 
   br: {
+    ...SimpleMarkdown.defaultRules.br,
     react: (_node, _output, state) => {
       return <Text key={state.key}>{"\n\n"}</Text>
```

Comment:
these were what was missing, none of the above would actually render and so we never showed br or newlines

Reply:
Whoa, nice catch.